### PR TITLE
refactor: add boime lint and fix lint errrors

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

- [x] 增加 boime lint 校验。
- [x] 修复 boime lint 相关错误。
- [x] precommit 中自动进行 boime check。
- [x] 优化部分组件的 a11y 问题。
- [x] 降低少许 bundlesize：340.24 KB (-181 B 🔽)
- [x] 移除 stylelint，用不上了。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    --       |
| 🇨🇳 Chinese |    --       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

**Comment by afc163** [comment]
收获 -104 B

File: components/_util/scrollTo.ts [hunk 1]
```diff
@@ -12,6 +11,11 @@ interface ScrollToOptions {
   duration?: number;
 }
 
+// Code From https://easings.net/#easeInOutCubic
+function easeInOutCubic(x: number): number {
+  return x < 0.5 ? 4 * x * x * x : 1 - (-2 * x + 2 ** 3) / 2;
```
Comment (li-jia-nan) [inline components/_util/scrollTo.ts:16]: 跟之前是不是不等价了？

Comment (afc163) [reply components/_util/scrollTo.ts]: 改回来了。

**Comment by afc163** [comment]
@SocketSecurity ignore-all

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**